satpy/tests/reader_tests/test_nucaps.py

```diff
@@ -24,10 +24,7 @@
 from satpy.tests.reader_tests.test_netcdf_utils import FakeNetCDF4FileHandler
 from satpy.tests.utils import convert_file_content_to_data_array
 
-if sys.version_info < (2, 7):
-    import unittest2 as unittest
-else:
-    import unittest
+import unittest
 
 try:
     from unittest import mock
```

Comment:
If you have to make other changes you could remove this try/except and only do `from unittest import mock`.

Reply:
Ah, yeah you mentioned that.